com.unity.cinemachine/Runtime/Behaviours/CinemachineGroupFraming.cs

```diff
@@ -116,23 +117,23 @@ void Reset()
             OrthoSizeRange = new Vector2(1, 1000);
         }
 
-        /// <summary>For editor visulaization of the calculated bounding box of the group</summary>
+        /// <summary>For editor visualization of the calculated bounding box of the group</summary>
         internal Bounds GroupBounds;
 
         /// <summary>For editor visualization of the calculated bounding box of the group</summary>
         internal Matrix4x4 GroupBoundsMatrix;
 
         class VcamExtraState
         {
-            public Vector3 m_PosAdjustment;
-            public Vector2 m_RotAdjustment;
-            public float m_FovAdjustment;
+            public Vector3 posAdjustment;
+            public Vector2 rotAdjustment;
+            public float fovAdjustment;
```

Comment:
should be PascalCase, not camelCase, if we're going to get picky about private structs